config/routes.rb

```diff
@@ -178,6 +178,10 @@
   post "/tags" => "tags#create"
   post "/tags/:tag_name" => "tags#update", :as => "update_tag"
 
+  dm_con = { :domain_name => /[-,0-z\.]+/ }
```

Comment:
I like that you broke out this constraint to share between the two routes. Can it also be shared with the existing `get "/domain/:name...` route? Maybe not - I've had constant hassle keeping that route happy with Rails wanting to interpret the `.com` as a mime type.

Reply:
It works well for .com domains. This constraint concern for this.
![constrains_for_com](https://user-images.githubusercontent.com/14085661/186888721-e899b16c-abde-4b5b-b1b1-3ba3e511d528.jpg)

Reply:
Yeah - I'm saying that in `config/routes.rb`, there's also lines 72 and 73 with basically this exact same constraint, and it would be good to reuse this so they all match exactly.

Reply:
Oh i see now. I will try to implement this constraint to it.